trusted-committer/06-advocating-for-the-communitys-needs.asciidoc

```diff
@@ -1,49 +1,31 @@
+They understand that a healthy community is a precondition for healthy code. 
+This is why many InnerSource initiatives were modeled in the Apache Way with it's http://theapacheway.com/community-over-code/["Community over Code"] motto. 
+Business units, on the other hand, are naturally more concerned with the products produced by an InnerSource community. 
+They prefer to see short- to medium-term results that help the bottom line.
 
-It is in this potential area of conflict where the Trusted Committer plays a vital role. Trusted Committers
-build trust with the organization and, building on that trust, act as an
-advocate for the interests of the community and the long term health of the
-software in the company. They are responsible for communicating technical as
-well community related risks to management. At the same time, Trusted Committers need to be
-strategic and work within the degrees of freedom afforded by their companies.
+It is in this potential area of conflict where the Trusted Committer plays a vital role. 
+Trusted Committers build trust with the organization and, building on that trust, act as advocates for the interests of the community and the long term health of the software in the company. 
+They are responsible for communicating technical, as well as community-related, risks to management.
+At the same time, Trusted Committers need to be strategic and work within the degrees of freedom afforded by their companies.
 
-Related to this, Trusted Committers need to make sure that the community and individual
-https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_contributors_] get public credit for their work, to make sure everyone is aware
-of the valuable contribution made. Public credit is kind of the currency in which
-contributors are being paid, especially those who contribute voluntarily. It is
-good practice to commend valuable contributors publicly and making sure their
-managers are CC'd as well. Neglecting to give credit, on the other hand, can be
-hugely frustrating for individual contributors and very detrimental for the
-health of the community overall. Neglecting to give credit can occur in
-companies which are not yet accustomed to the InnerSource working model or when
-the software being developed by the InnerSource community runs _behind the
-scenes_ and managers were simply not aware of the community's contribution. A
-good Trusted Committer will engage with management and advocate the need for
-public credit in this case. Failure to give credit is almost never done in bad
-faith, though, and Trusted Committers should be able to easily correct that.
+Trusted Committers also need to make sure that the community and individual https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_contributors_] get public credit for their work. 
```

Comment:
@tsadler1988 Do you have an idea how inter-article linking would work with the straight-from-GitHub rendering you made? 
Maybe use some identifier from our mapping JSON here (e.g. as non-existing-anchor in the link here) and fetch the relevant rendered URL? (I suck at JS and hence can only come up with horrible, totally not state-of-the-art hacks like that :wink: )

Reply:
As things stand, that link will just go to GitHub. I believe this was raised and we accepted this for the first pass.

An identifier would be ideal, but doing a string replace of each literal GitHub link to the equivalent innersourcecommons.org link should be feasible. I will try and have something to show on this by Thursday :)

Reply:
> As things stand, that link will just go to GitHub. I believe this was raised and we accepted this for the first pass.

Sorry, I forgot that. My bad. 

> 
> An identifier would be ideal, but doing a string replace of each literal GitHub link to the equivalent innersourcecommons.org link should be feasible. I will try and have something to show on this by Thursday :)

W00t w00t! :tada:

Reply:
Do we have multiple final locations where we'd like these links to point?  Or will they just point to the ISC website each time?

Reply:
@rrrutledge are you suggesting that the `.asciidoc` files' links should always point to `innersourcecommons.org`? That seems reasonable, but could this mean less reason to keep the `urls.yaml` up to date? How would that be enforced?

Reply:
Yes, I am suggesting that.

Delete `urls.yaml`.

Discussion in this week's call?

Reply:
I'm just casually throwing in here #244 as a possible second publication target.

Reply:
ok to merge according to [last meeting notes](https://docs.google.com/document/d/1ktfIb4-5u9AmAC1dB9LhbjquuopijY1DVcN5JdFpPPs/edit)